src/RbacMigrateTrait.php

```diff
@@ -261,8 +261,9 @@ private function createBaseMigrationClass()
     private function ensureRuleClassExists($className, $name)
     {
         try {
-            class_exists($className, true);
-            return;
+            if (class_exists($className, true)) {
+                return;
+            }
         } catch (UnknownClassException $e) {
             // fine, we are working on
         }
```

Comment:
Maybe we even not needed in this try-catch?

Reply:
@erickskrauch yes, updated.